src/ui/public/vis_maps/visualizations/_map.js

```diff
@@ -60,10 +61,17 @@ export default function MapFactory(Private, tilemapSettings, Notifier) {
         fadeAnimation: false,
       };
 
+      let url;
       if (tilemapSettings.hasError()) {
-        notify.fatal(tilemapSettings.getError());
+        if (!previousErrors.includes(tilemapSettings.getError())) {
+          notify.warning(tilemapSettings.getError().message);
+          previousErrors.push(tilemapSettings.getError());
+        }
+        url = '';
+      } else {
+        url = tilemapSettings.getUrl();
       }
-      this._createMap(options, tilemapSettings.getUrl());
+      this._createMap(options, url);
```

Comment:
I don't know if we should be creating the map when there's an error. I was able to create a fatal error by using the map that was rendered after the tilemap-proxy is unavailable.